library/posture.c

```diff
@@ -183,13 +185,24 @@ void ziti_send_posture_data(ziti_context ztx) {
     ZTX_LOG(VERBOSE, "starting to send posture data");
     bool new_session_id = ztx->posture_checks->previous_session_id == NULL || strcmp(ztx->posture_checks->previous_session_id, ztx->api_session->id) != 0;
 
-    if (new_session_id || ztx->posture_checks->must_send_every_time) {
-        ZTX_LOG(DEBUG, "posture checks either never sent or api session changed, must_send = true");
+    bool new_controller_instance = (ztx->posture_checks->controller_instance_id == NULL && ztx->controller.instance_id != NULL) || strcmp(ztx->posture_checks->controller_instance_id, ztx->controller.instance_id) != 0;
```

Comment:
could you please output a log when this boolean becomes only  true ?

Reply:
done